src/components/top-bar-equipment-seach-dialog/top-bar-equipment-search-input.tsx

```diff
@@ -39,22 +39,26 @@ export const TopBarEquipmentSearchInput = (
 ) => {
     const {
         displayedValue,
-        isSearchDisabled,
         params,
         equipmentType,
         setEquipmentType,
+        disabledSearchReason,
```

Comment:
we don't need nor `isSearchDisabled` nor `disabledSearchReason`.
`disabledSearchReason` is contained in `displayedValue`
`isSearchDisabled` use should be changed by `params.disabled`

Reply:
Done.